manifest/manifest.go

```diff
@@ -124,3 +125,9 @@ func ParseDir(chartDir string) ([]*Manifest, error) {
 
 	return files, filepath.Walk(dir, walker)
 }
+
+// IsKeeper returns true if a "helm-keep: true" annotation is present in manifest bytes.
+func IsKeeper(data []byte) bool {
+	var keep = regexp.MustCompile(`\"annotations\": \{(\n.*){0,10}\"helm-keep\": true`)
```

Comment:
Maybe this would be a reason to revisit the idea of putting this in the file header instead of as an annotation? That didn't look quite as straightforward to implement, but could be safer to parse out.